### PR TITLE
users#myorders. added table for myorders page, added "ship" functionality and tests

### PR DESCRIPTION
**Pull request by aribray**
# bEtsy
Congratulations! You're submitting your assignment! These comprehension questions should be answered by all members of your team, not by a single teammate.

## Comprehension Questions
Question | Answer
:------------- | :-------------
Each team member: what is one thing you were primarily responsible for that you're proud of? | 
Each team member: what is one thing you were primarily responsible for that you would like targeted feedback on? | 
How did your team break up the work to be done? | 
How did your team utilize git to collaborate? | 
What did your group do to try to keep your code DRY while many people collaborated on it? | 
What was a technical challenge that you faced as a group? | 
What was a team/personal challenge that you faced as a group? | 
What was your application's ERD? (include a link) | 
What is your Trello URL? | 
What is the Heroku URL of your deployed application? |

### CODE REVIEW

File: test/controllers/orderitems_controller_test.rb [hunk 1]
```diff
@@ -56,16 +59,49 @@
...
+      end.wont_change 'Product.count'
 
       must_respond_with :redirect
     end
   end
+
+  describe 'ship' do
+    describe 'logged in users' do
+      it 'will change orderitems shipped status to true' do
+        perform_login
```
Comment (minams) [inline test/controllers/orderitems_controller_test.rb]: nothing major but i think we can pull the perform _login before the 'describe "logged in users" do' and then we don't have to perform login each time.

File: app/views/users/myaccount.html.erb [hunk 2]
```diff
@@ -1,12 +1,14 @@
-
+<%@user = User.first%>
```
Comment (minams) [inline app/views/users/myaccount.html.erb]: :)

**Review by minams — CHANGES_REQUESTED** [review]
I am in awe of your smartness.  Seriously.  Great job!  Just a few suggestions!

File: app/views/users/myaccount.html.erb (hunk 2 above)
Comment (aribray) [reply app/views/users/myaccount.html.erb]: teehee

File: test/controllers/orderitems_controller_test.rb (hunk 1 above)
Comment (aribray) [reply test/controllers/orderitems_controller_test.rb]: done!

**Review by minams — APPROVED** [review]
10s 10s 10s across the board!!!

**Pull request merged**